elenacaseyroby/YoutubeMusicLib
# added sql commands to create db and a class for each table in the model

# What

Added sql commands to create db and a class for each table in the model
# Why

So Youtube plays data can be stored 
# Test
1. post to db from python console

```
>>> from app import db, models
>>> from sqlalchemy.orm import sessionmaker
>>> Session = sessionmaker(bind=models.engine)
>>> Session = sessionmaker()
>>> Session.configure(bind=models.engine) 
>>> session = Session()
>>> session.rollback()
>>> casey_user = models.User(email="elenacaseyroby@gmail.com", first_name="Casey", last_name="Roby", verification_level=100);
>>> session.add(casey_user)
>>> session.commit()
```
1. Check to see if the data was posted to the db:
   <img width="501" alt="screen shot 2016-06-28 at 12 42 49 pm" src="https://cloud.githubusercontent.com/assets/11428059/16430846/ad2b7a3c-3d4b-11e6-97a6-ccc619d02c74.png">
2. Run server and perform two music searches at http://localhost:5000/index to make sure we haven't broken anything else:

```
python run.py
```
1. If all is running smoothly, merge it!
